### PR TITLE
Drop --no-audit workaround.

### PR DESCRIPTION
**Pull request by mshima**
Original issue is fixed https://github.com/jhipster/generator-jhipster/pull/14315#issuecomment-844149669

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/server/index.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
           }
           const gradle = this.jhipsterConfig.buildTool === GRADLE;
           const command = gradle ? './gradlew' : './npmw';
-          const args = gradle ? ['npm_install'] : ['install', '--no-audit'];
+          const args = gradle ? ['npm_install'] : ['install'];
```
Comment (atomfrede) [inline generators/server/index.js:66]: Should we switch back to `npmInstall`?
```suggestion
          const args = gradle ? ['npmInstall'] : ['install'];
```

Comment (mshima) [reply generators/server/index.js:66]: You are the expert.

**Comment by pascalgrimaud** [comment]
@mshima : I didn't follow this one, but is it safe to be merged ?

**Comment by mshima** [comment]
NPM was failing when install was executed due to a bug in audit.
It’s fixed now.
Yes I think it’s safe.

**Pull request merged**